client/src/components/Dashboard-FormAssemply.js

```diff
@@ -0,0 +1,27 @@
+import React, { Component } from 'react';
+import * as S from './Dashboard-FormAssemply.style';
+import * as C from './Journal.style';
+
+class FormAssemply extends Component {
```

Comment:
also here, p turns to b - ```FormAssembly```

also, does this need to be a class? or since it is not stateful, can we convert it to a function?

Reply:
it should be a class for whoever will be working on it